src/server/middleware/csp.js

```diff
@@ -69,7 +69,10 @@ const csp = () => (req, res, next) => {
   }
 
   res.scriptNonce = scriptNonce;
-  res.setHeader('Content-Security-Policy', updatedPolicy);
+
+  if (!process.env.ONE_DANGER_DISABLE_CSP) {
```

Comment:
Shouldn't we also check NODE_ENV ? The way it is currently CSP can be disabled in prod

Reply:
I put it as such because if the environment variable is present but NODE_ENV is not in dev the runner won't continue past the second check in checkCSP

Reply:
Consider separating `checkCsp` into a util function that can be reused here